templates/parcel_new.html

```diff
@@ -28,10 +28,10 @@
       });
 
       $('#lot').on('change', function() {
-        var idLot = $(this).val()
+        var lot_id = $(this).val()
         var delivery_type = $('#delivery').val()
-        $.getJSON('/pick_products', {
-          id: idLot,
+        $.getJSON($(this).attr('data-href'), {
```

Comment:
`data-*` attributes can be accessed through the [`data()` method](https://api.jquery.com/data/#data2)